landlab/components/flow_director/tests/test_flow_director.py

```diff
@@ -379,3 +379,18 @@ def test_change_bc_post_init():
         ]
     )
     assert_array_equal(new_true_reciever, fd.receiver)
+
+
+def test_link_flow_direction_hex():
```

Comment:
Are you just trying to pad our testing stats?

Reply:
nope! I think that was an old test that I outlined but never wrote... removing.